extensions/transmission/src/addNewTorrent.tsx

```diff
@@ -84,17 +88,20 @@ export default function AddNewTorrent() {
       <Form.TextField
         title="Magnet Link or Torrent File"
         id="input"
-        placeholder="magnet: (or ⌘ O to select from Finder)"
+        placeholder="magnet:..."
         value={input}
         onChange={(value) => setInput(value)}
       />
+      <Form.Description text="⌘ O to select from Finder" />
```

Comment:
It does, but that would prevent magnet links from being used. 

It would be great to overhaul this form with the file picker OR a field for the magnet link, fix the clipboard detection, and maybe improve the UX and validation of the file path, but I didn't have time to do that yesterday. 

If this bothers me I might try to add it soon, but let's consider these changes as is for now and try to get a new release out if you're happy with them.